tools/translation/ignore_incomplete.sh

```diff
@@ -0,0 +1,69 @@
+#! /usr/bin/env bash
+
+# ===========================================================================
+#
+# Copyright (c) 2024 Unvanquished Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+# ===========================================================================
+
+set -u -e -o pipefail
```

Comment:
Does the script only add things to the ignore list, or can it also un-ignore them?

Reply:
It also un-ignore them:

1. Read every lines from the action file that do not start with `ignore translation/`:

```sh
action_list="$(grep -v -E '^ignore translation/' "${action_file}")"
```

2. Rewrite the action file without any `ignore translation/*` lines:

```sh
printf '%s\n' "${action_list}" > "${action_file}"
```

3. Add (or readd) to the action file a new `ignore` command for a translation file:

```sh
printf 'ignore translation/%s\n' "${po_file}" >> "${action_file}"
```

Reply:
We concluded that some languages such as Arabic may not be ready even at 100%. Maybe it shouldn't remove things?

Reply:
Right! With luck urcheon has to way to ignore things, an `ignore.txt` file that acts a bit like `.gitignore` (it supports some globbing), and the action file. The `ignore.txt` is purposed more for things like ignoring a whole directory, that's why the script updates the action file instead, which is only purposed to have a single command listed per single file. We can add the arabic file to the `ignore.txt` and then keep the script as simple as it is today.

Reply:
Now done. I also forgot that the `ignore.txt` file doesn't support complete paths (that's why the action list is better for ignoring a precise file), so I marked any path of `ar.po` to not be packaged.